pkg/cli/node.go

```diff
@@ -316,8 +316,12 @@ func runDecommissionNode(cmd *cobra.Command, args []string) error {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	if nodeCtx.nodeDecommissionSelf {
+		fmt.Fprintf(stderr, "warning: --%s is deprecated.\n", cliflags.NodeDecommissionSelf.Name)
```

Comment:
[nit] Maybe just a few more words here too about how we're going to skip the drain step.

Reply:
That warning happens here so that this warning is also sent in the more general case where a target node is the host:

```go 
// We now drain the nodes in order to close all SQL connections.
// Note: iteration is not necessary here since there are no remaining leases
// on the decommissioning node after replica transferral.
for _, targetNode := range nodeIDs {
  if targetNode == localNodeID {
    // Skip the draining step for the node serving the request, if it is a target node.
    log.Warning(ctx,
      "node specified with --host should not be a target node; skipped the drain step for this node.",
    )
    continue
  }
  .
  .
  .
}